src/bp/ui-studio/src/web/components/Layout/StatusBar.jsx

```diff
@@ -124,13 +127,33 @@ export default class StatusBar extends React.Component {
     ))
   }
 
+  renderDocHints() {
+    if (!this.props.docHints.length) {
+      return null
+    }
+
+    const onClick = () => this.props.updateDocumentationModal(this.props.docHints[0])
```

Comment:
should be declared outside render, since each time the status bar is rendered, a new instance is created